zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/NoteManager.java

```diff
@@ -78,8 +78,8 @@ private void init() throws IOException {
         {
           try {
             addOrUpdateNoteNode(new Note(new NoteInfo(entry.getKey(), entry.getValue())));
-          } catch (IOException e) {
-            LOGGER.warn(e.getMessage());
+          } catch (Throwable e) {
+            LOGGER.warn("addOrUpdateNoteNode Fail: {}", entry.getKey() + " - " + e.getMessage(), e);
```

Comment:
I think it is better to catch the exception and log the warning when invoking NotebookRepos#get

Reply:
And please rebase your PR first.

Reply:
> I think it is better to catch the exception and log the warning when invoking NotebookRepos#get

In my case, Throwed exception that line when I started zeppelin, not open notebook.

I checked `addOrUpdateNoteNode` method, It isn't use `NotebookRepos#get`.

Why do you think that?

Reply:
Could you paste the stacktrace? It is weird how damaged notebook will happen if no reading is invoked.

Reply:
Unfortunately, I cannot get stacktrace, anymore because I moved company. 

I think that maybe filenames or folder names  were weird.

When I saw that exception, I tried to upgrade 0.7 to 0.9. So I ran script for upgrade notebooks. Many staff of company used zeppelin (not only data team), many notebook's name could be bad. (even use `*`, `(`, `)`,`[`, `]`, etc)

Reply:
I see, could you add one comment to explain that the invalid file name would cause exception here?